app/helpers/application_helper.rb

```diff
@@ -382,6 +382,19 @@ def generate_links_for(sub_model)
     sub_model.map {|model| link_to(model.to_label, { :controller => model.class.model_name.plural.downcase, :action => :index, :search => "name = \"#{model.name}\"" })}.to_sentence
   end
 
+  def editable_select_optgroup(groups, options = {})
```

Comment:
this helper method generates dynamic `optgroups` in editable select

![screenshot from 2016-07-28 15-54-06](https://cloud.githubusercontent.com/assets/11807069/17233086/8d1c0a3e-5535-11e6-9790-a70afd36d07c.png)

`groups` array includes hashes, each hash represent a distinct group.